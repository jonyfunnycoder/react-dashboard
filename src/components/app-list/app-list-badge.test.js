import React from 'react';
import renderer from "react-test-renderer";
import AppListBage from "./app-list-badge";

test('renders app list badge correctly ', () => {
    const tree = renderer
        .create(<AppListBage value="567567" />)
        .toJSON();

    expect(tree).toMatchSnapshot();
});
