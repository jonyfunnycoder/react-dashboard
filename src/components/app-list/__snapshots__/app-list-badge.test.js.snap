// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app list badge correctly  1`] = `
<div
  className="list-badge"
>
  567567
</div>
`;
